src/components/Geolocation.tsx

```diff
@@ -0,0 +1,124 @@
+import { Matcher, SelectableStaticFilter, useSearchActions, useSearchState } from '@yext/search-headless-react';
+import { executeSearch } from '../utils/search-operations';
+import { getUserLocation } from '../utils/location-operations';
+import { useComposedCssClasses } from '../hooks/useComposedCssClasses';
+import { useCallback, useState } from 'react';
+import LoadingIndicator from '../icons/LoadingIndicator';
+import { YextIcon } from '../icons/YextIcon';
+
+/**
+ * The CSS class interface for the Geolocation component.
```

Comment:
I cannot use `{@link Geolocation}` here due to an issue with API Extractor adding `_2` suffix to duplicate interface/class names. In search-ui-react, [Geolocation from Typescript](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.geolocation.html)'s "lib.dom.d.ts" file conflicts with Geolocation component.
See: https://github.com/microsoft/rushstack/issues/2534 and https://github.com/microsoft/rushstack/issues/2976
Open PR: https://github.com/microsoft/rushstack/pull/2608

<img width="300" alt="Screen Shot 2022-11-16 at 12 15 08 PM" src="https://user-images.githubusercontent.com/36055303/202248270-d49df07f-01ce-4a4a-8b48-b41ac05093ce.png">

What to know if the team have any thoughts on this. A work around is to add a script to manually replace naming in the files generated from API extractor, which is hacky.

Reply:
I'm fine with it, the api extractor docs aren't made to be extremely beginner friendly

Reply:
could we link to `Geolocation_2`? or would that not resolve correctly?

Reply:
it would link correctly in the doc pages but we would see this in search-ui-react.api.md file:
<img width="593" alt="Screen Shot 2022-11-16 at 3 28 29 PM" src="https://user-images.githubusercontent.com/36055303/202287297-8f9adcc4-8c7e-4d97-8b89-504fdee88057.png">

Reply:
oh ok, I'm fine leaving it without a link